src/Symfony/Component/Console/Style/SymfonyStyle.php

```diff
@@ -183,6 +183,8 @@ public function caution($message)
      */
     public function table(array $headers, array $rows)
     {
+        $headers = array_map(function ($value) { return sprintf('<info>%s</>', $value); }, $headers);
```

Comment:
I personally use it because the code looks more much concise and clear to me. Since this is just markup and not "real code", I want it to mess my code as less as possible. That's why using `</info>` looks very repetitive to me. When using custom styles (e.g. `<fg=blue>`) duplicating it in the closing tag loks awfully boring to me.